build.gradle

```diff
@@ -30,7 +30,7 @@ dependencies {
 	shadow group: 'org.bstats', name: 'bstats-bukkit', version: '3.0.2'
 	shadow group: 'net.kyori', name: 'adventure-text-serializer-bungeecord', version: '4.3.2'
 
-	implementation group: 'io.papermc.paper', name: 'paper-api', version: '1.21-R0.1-SNAPSHOT'
+	implementation group: 'io.papermc.paper', name: 'paper-api', version: '1.21.1-R0.1-SNAPSHOT'
```

Comment:
last i checked, 1 is not equal to 3

Reply:
Like I said, I haven't pushed changes yet. There is no dev server bundle available yet. Once that is I will push the changes.

Reply:
Oh please don't update the pr saying it does something it doesn't yet do, then

Reply:
Sorry, will keep note of that in the future.